package/MDAnalysis/analysis/psa.py

```diff
@@ -405,6 +463,13 @@ def hausdorff_wavg(P, Q):
      2.5669644353703447
      >>> hausdorff_wavg(P,Q[::-1]) # weighted avg hausdorff dist w/ Q reversed
      2.5669644353703447
+
+    Notes
```

Comment:
Yes, its `Notes` as in https://github.com/numpy/numpy/blob/master/doc/example.py